hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -2673,6 +2677,24 @@ String reconfigureSlowNodesParameters(final DatanodeManager datanodeManager,
         datanodeManager.setMaxSlowPeersToReport(maxSlowPeersToReport);
         break;
       }
+      case DFS_NAMENODE_SLOWPEER_COLLECT_INTERVAL_KEY: {
+        if (newVal == null) {
+          // set to the value of the current system or default
+          long defaultInterval =
+              getConf().getTimeDuration(DFS_NAMENODE_SLOWPEER_COLLECT_INTERVAL_KEY,
+                  DFS_NAMENODE_SLOWPEER_COLLECT_INTERVAL_DEFAULT, TimeUnit.MILLISECONDS);
+          datanodeManager.restartSlowPeerCollector(defaultInterval);
+          result = DFS_NAMENODE_SLOWPEER_COLLECT_INTERVAL_DEFAULT;
```

Comment:
Maybe this is the correct line?
```suggestion
          result = Long.toString(defaultInterval);
```

Reply:
> Maybe this is the correct line?

Thanks, I have fixed it.